src/main/java/com/fasterxml/jackson/databind/util/EnumResolver.java

```diff
@@ -76,7 +76,7 @@ public static EnumResolver constructUsingToString(Class<Enum<?>> enumCls,
         // from last to first, so that in case of duplicate values, first wins
         for (int i = enumValues.length; --i >= 0; ) {
             Enum<?> e = enumValues[i];
-            map.put(e.toString(), e);
+            map.put(e.toString() + "", e);
```

Comment:
or maybe it should continue if e is null and not add anything to the map

Reply:
I think I like "skip nulls" approach: will use that.